quantlib/handle.pxd

```diff
@@ -16,11 +16,12 @@ cdef extern from 'ql/handle.hpp' namespace 'QuantLib':
         Handle(T*)
```

Comment:
Yes, that's true. They are a number of warnings when building the library. This should be cleaned up but will require some efforts